content/eng/central/commerce_and_industries/AIC.md

```diff
@@ -0,0 +1,68 @@
+
+#### Expected Functions of the AICs:
+
+    1. Assist the incubatees in creating a sustainable, scalable and profitable business model.
+    2. Provide physical infrastructure and value-added support services.
+    3. Create a strong network of mentors who would provide sector specific knowledge and real world practical guidance.
+    4. Conduct inspirational programs and events.
+    5. Provide trainings and mentorship to entrepreneurs.
+    6. Forge partnerships and networks with academia, industry, funding sources, existing incubators and other for the start-ups to leverage.
+    7. Enable access to prototyping facilities, test beds, markets and pilot implementation for the product/services.
+    8. Build a strong team with adequate knowledge and experience in guiding start-ups, building business plans, facilitating investments, building networks etc.
+    9. An ideal application would be a collaboration between a corporate sector entity and a research oriented/ academic institution with aligned area of focus.
+
+#### Features of the Scheme:
+
+    1. Keeping in view the learnings from round 1 of AIC selections in 2016-2017, it has been decided to broaden the applicant pool, by inviting a wider category of entities, such as higher educational institutions, R&D institutes, corporate sector, alternative investment funds registered with SEBI, business accelerators, group of individuals, and individuals.
+    2. AICs can be established by one or more collaborating entities. For the purposes of this application, only the entity who will be providing the space for setting up the AIC shall be considered as the applicant.
+    3. AICs are encouraged to focus on a limited number of areas, which align with their core strengths. The AIC should have upto a maximum of three primary sectors of focus.
+    4. Successful AIC applicants will be required to incorporate a Special Purpose Vehicle(SPV), either a Section 8 company or a private limited company registered under the Companies Act, 2013. Based on learnings from round 1 of AICs, these SPVs will not be allowed to incorporate as trusts and societies.
+
+#### Funding Support:
+
+    1. Successful applicants would be provided grant-in-aid to set up the AIC for a maximum period of 5 years upto Rs. 10 crore to cover the capital and operational expenditures. The exact amount of the grant shall be decided by AIM based on the application and business plan, as per the prevailing scheme. AIM reserves the right to modify the scheme as and when required, without prior notifications.
```

Comment:
create a hover thing on acronym like AIC or AIM so that full form will appear, it will be easier for people to remember . Just check if it can easily done in markdown

Reply:
there are multiple reference to AIM and AIC. So it doesn't look good at every place